language/automl/automl_natural_language_dataset.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to perform basic operations on Dataset
+with the Google AutoML Natural Language API.
+
+For more information, see the tutorial page at
+https://cloud.google.com/natural-language/automl/docs/
+"""
+
+# [START automl_natural_language_import]

```

Comment:
Do we need region tags for imports too? I did not know.

Reply:
removed